scripts/checkstyle_lint_check_github_action.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+echo "********************************"
+echo "Checking Java file formatting"
+echo "********************************"
+
+lint_results=$(java -jar checkstyle-8.37-all.jar -c /google_checks.xml app/src/ data/src/ domain/src/ utility/src/ testing/src/ 2>&1)
```

Comment:
creating this separate file because for pre-push we need `../oppia-android-tools/`.

To remove this duplicacy of the file, with we can change the path of pre-push files!!

Reply:
Adding @Srikanth-Kb for any thoughts on this.

Reply:
Hmm, I was expecting to have a single script rather than 2. Why doesn't oppia-android-tools work in the GitHub action world?

Reply:
Continuing this discussion. on the newer comment thread below.